systemtest/src/test/java/io/strimzi/systemtest/kafka/dynamicconfiguration/DynamicConfigurationSharedST.java

```diff
@@ -179,6 +186,41 @@
         return testCases;
     }
 
+    /**
+     * Method, which randomly choose 3 dynamic properties for verification from @see{testCases}. In this case we are ok
+     * with stochastic selection, because we don't care, which configuration is used. Furthermore, it's the same path
+     * of code (i.e., same CFG (control flow graph), which does not triggers RollingUpdate).
+     * @param testCases test cases, where each consist of one dynamic property
+     * @return List 3 chosen dynamic properties
+     */
+    private static List<String> stochasticSelection(final Map<String, Object> testCases) {
+        final List<String> testCaseKeys = new ArrayList<>(testCases.keySet());
+        final Random generator = new Random();
+        final List<String> chosenDynConfigurations = new ArrayList<>(3);
+
+        for (int i = 0; i < 3; i++) {
+            int stochasticNumber = generator.nextInt(testCaseKeys.size());
+            String chosenDynConfiguration = testCaseKeys.get(stochasticNumber);
+
+            if (chosenDynConfigurations.contains(chosenDynConfiguration)) {
+                while (chosenDynConfigurations.contains(chosenDynConfiguration)) {
+                    LOGGER.debug("List of dynamic configurations already have that type of configuration:\n{}", chosenDynConfigurations.toArray());
+                    stochasticNumber = generator.nextInt(testCaseKeys.size());
+                    chosenDynConfiguration = testCaseKeys.get(stochasticNumber);
+
+                    chosenDynConfigurations.add(chosenDynConfiguration);
```

Comment:
So you are sure you won't pick the same (or already present) key twice?

Reply:
I am 100% sure, but its greedy algorithm (which is not good), because it can happen that will iterate over the collection of Dyn. Confs and pick the same twice (but it will not add to the list), which results in another iteration in a while loop. Re-writed to:
```
final int stochasticNumber = rng.nextInt(testCaseKeys.size());
final String chosenDynConfiguration = testCaseKeys.get(stochasticNumber);
LOGGER.debug("New configuration in List of dynamic configuration:\n{}", chosenDynConfigurations.toArray());
chosenDynConfigurations.add(chosenDynConfiguration);
// remove it from `testCaseKeys` list to not include it twice
testCaseKeys.remove(chosenDynConfiguration);
```